### PR TITLE
Beacon id field

### PR DESCRIPTION
**Pull request by mrmike**
* Added `beacon_id` field for `mobile_app_tracking_event`
* Deployed to dev environment

### CODE REVIEW

File: service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java [hunk 1]
```diff
@@ -61,7 +62,7 @@ public void toSimpleStringWithEmptyGeoIpData() {
         .map(new MobileAppDataToSimpleString())
         .findFirst().get();
     assertEquals(
-        "192.168.1.1|app_name=x&app_version=1.1&device_id=12&timestamp=1234&user_id=1|||||||||referer",
+        "192.168.1.1|app_name=x&app_version=1.1&device_id=12&timestamp=1234&user_id=1&beacon_id=add743|||||||||referer",
```
Comment (WikiaTech) [inline service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java]: MAJOR Make this IP "192.168.1.1" address configurable. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1313)

File: service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java [hunk 2]
```diff
@@ -91,7 +92,7 @@ public void toSimpleStringMostData() {
         .map(new MobileAppDataToSimpleString())
         .findFirst().get();
     assertEquals(
-        "192.168.1.1|app_name=x&app_version=1.1&device_id=12&timestamp=1234&user_id=1&ga.category=category&"
+        "192.168.1.1|app_name=x&app_version=1.1&device_id=12&timestamp=1234&user_id=1&beacon_id=add743&ga.category=category&"
```
Comment (WikiaTech) [inline service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java]: MAJOR Make this IP "192.168.1.1" address configurable. [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1313)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 4 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Make this IP "192.168.1.1" address configurable.](https://github.com/Wikia/pandora/blob/cfe0f6ecfaeb3fbacfcaf24d9425859e14ef0a99/service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java#L40) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1313)
* MAJOR [Make this IP "192.168.1.1" address configurable.](https://github.com/Wikia/pandora/blob/cfe0f6ecfaeb3fbacfcaf24d9425859e14ef0a99/service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java#L84) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1313)

**Comment by pchojnacki** [comment]
LGTM

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues:
* MAJOR 4 major

Watch the comments in this conversation to review them.
Note: the following issues could not be reported as comments because they are located on lines that are not displayed in this pull request:
* MAJOR [Make this IP "192.168.1.1" address configurable.](https://github.com/Wikia/pandora/blob/cfe0f6ecfaeb3fbacfcaf24d9425859e14ef0a99/service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java#L40) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1313)
* MAJOR [Make this IP "192.168.1.1" address configurable.](https://github.com/Wikia/pandora/blob/cfe0f6ecfaeb3fbacfcaf24d9425859e14ef0a99/service/clickstream/src/test/java/com/wikia/service/clickstream/serializer/SerializationTest.java#L84) [rule](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1313)

**Pull request merged**